glossary/UNIVERSAL.md

```diff
@@ -0,0 +1,7 @@
+# Universal
+
+A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
```

Comment:
Ok, It should be done now.

On 6 December 2015 at 21:06, Hugo Giraudel <notifications@github.com> wrote:

> In glossary/UNIVERSAL.md
> <https://github.com/HugoGiraudel/SJSJ/pull/46#discussion_r46775117>:
>
> > @@ -0,0 +1,7 @@
> > +# Universal
> > +
> > +A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
>
> If you don’t mind. Else I’ll do it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HugoGiraudel/SJSJ/pull/46/files#r46775117>.
>



-- 
Kind regards,

*- Robert Smith*
*Web Developer*

-----

*W*: http://rbrtsmith.com/
*T*: +46(0)73 687 5773
*E*: rbrtsmith@googlemail.com